src/status_im2/contexts/wallet/common/utils.cljs

```diff
@@ -43,3 +43,15 @@
               (* (total-token-value-in-all-chains token)
                  (-> token :market-values-per-currency :usd :price))))
        (reduce +)))
+
+(defn calculate-fiat-change
+  [fiat-value change-pct-24hour]
+  (* fiat-value (/ change-pct-24hour (+ 100 change-pct-24hour))))
```

Comment:
I'm not sure if these calculations are correct, but we should be using bignumbers, it's highly probable that `fiat-value` here is wrong.

It's worth checking the `utils/money` ns. Maybe there's an existing function that does this.

Reply:
Thanks for the heads up @ulisesmac! Changed to use `bignumber`.

I checked `utils/money` ns but there does not seem to be a function to calculate the fiat change from a value and a percentage.